app/components/homepage-components/Hero.tsx

```diff
@@ -35,16 +36,18 @@ const Hero = () => {
           digital experiences with precision, passion and a profound commitment
           to excellence.
         </p>
-        <div className="mt-9 flex flex-col md:flex-row md:justify-between md:gap-4">
-          <Button
-            type="blueButton"
-            additionalStyles="w-full py-3.5 text-sm md:w-[15rem] md:py-5 md:text-lg"
-          >
-            My Work
-          </Button>
+        <div className="mt-9 flex flex-col md:flex-row md:gap-4">
+          <Link href="/case-studies">
+            <Button
+              type="blueButton"
```

Comment:
Generally, a button only has three different types.
submit  — Submits the current form data. (This is the default.)
reset  —    Resets data in the current form.
button —  Just a button. Clickable button; Its effects must be controlled by something else (that is, with [W3](https://www.w3schools.com/tags/att_button_type.asp).


Read more: https://html.com/attributes/button-type/#ixzz8DQWoklIs